tests/e2e/e2e_test.go

```diff
@@ -232,6 +254,95 @@ var _ = ginkgo.Describe("[Start/Remove/Restart/Add/Stop]", func() {
 			gomega.Ω(err).Should(gomega.BeNil())
 		})
 
+		ginkgo.By("can create a blockchain with new subnet id with some of existing participating nodes", func() {
+			ux.Print(log, logging.Blue.Wrap("can create a blockchain in an existing subnet"))
+			ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
+			_, err := cli.CreateBlockchains(ctx,
+				[]*rpcpb.BlockchainSpec{
+					{
+						VmName:     "subnetevm",
+						Genesis:    "tests/e2e/subnet-evm-genesis.json",
+						SubnetSpec: &rpcpb.SubnetSpec{Participants: subnetParticipants},
+					},
+				},
+			)
+			cancel()
+			gomega.Ω(err).Should(gomega.BeNil())
+		})
+
+		ginkgo.By("wait for custom chains healthy", func() {
+			ctx, cancel := context.WithTimeout(context.Background(), 2*time.Minute)
+			_, err := cli.WaitForHealthy(ctx)
+			cancel()
+			gomega.Ω(err).Should(gomega.BeNil())
+		})
+
+		ginkgo.By("verify subnet has correct participants", func() {
+			ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
```

Comment:
can you check that the new blockchain is associated to this subnet? probably look up for the new blockchain and then look at for the participants on it